cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java

```diff
@@ -123,7 +124,7 @@ public static void pushPackages(String packageName, String sourceRoot, String in
             // Push package to central
             String resourcePath = resolvePkgPathInRemoteRepo(packageID);
             String msg = orgName + "/" + packageName + ":" + version + " [project repo -> central]";
-            Proxy proxy = RepoUtils.readSettings().getProxy();
+            Proxy proxy = settings.getProxy();
```

Comment:
Can this (settings) never be null?

Reply:
Initialized the settings object when defining it with 889afe6. This will ensure that it never becomes null